srcpkgs/vosk-api/template

```diff
@@ -0,0 +1,78 @@
+# Template file for 'vosk-api'
+pkgname=vosk-api
+version=0.3.43
+revision=1
+_openblas_ver=0.3.20
+_clapack_ver=3.2.1
+_openfst_commit=7dfd808194105162f20084bb4d8e4ee4b65266d5
+_kaldi_commit=6417ac1dece94783e80dfbac0148604685d27579
```

Comment:
could it work with system packages?

Reply:
Thanks paper42.
I'd forgotten to remove the openblas and clapack variables, which have been replaced with system packages.
The version of kaldi is vosk specific so I think it's best as it is.
I tried packaging openfst separately but the newer version wasn't compatible (and there's vosk specific changes but I didn't get that far).